app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -323,9 +323,8 @@ class ComposeViewModel @Inject constructor(
         serviceClient.sendToot(tootToSend)
     }
 
-    // Updates a QueuedMedia item arbitrarily, then sends description and focus to server
-    private suspend fun updateMediaItem(localId: Int, mutator: (QueuedMedia) -> QueuedMedia): Boolean {
-        val newMediaList = media.updateAndGet { mediaValue ->
+    private fun updateMediaItem(localId: Int, mutator: (QueuedMedia) -> QueuedMedia) {
+        media.updateAndGet { mediaValue ->
```

Comment:
I have the feeling that this could be simplified even much further? Get rid of updateMediaItem and "updateAndGet"?

(Simply set the respective field - with a simple "find" before?)

Reply:
Probably not: This flow (its list) is observed by the activity.

However I made it a bit more clear: "AndGet" was never necessary here and "mediaValue" is actually a "mediaList".